Doc/c-api/unicode.rst

```diff
@@ -987,15 +993,21 @@ These are the UTF-8 codec APIs:
    .. versionchanged:: 3.10
       This function is a part of the :ref:`limited API <limited-c-api>`.
 
+   .. versionchanged:: 3.13
+      Raise an exception if *size* is NULL and the string contains embedded
+      null characters.
+
 
 .. c:function:: const char* PyUnicode_AsUTF8(PyObject *unicode)
 
-   As :c:func:`PyUnicode_AsUTF8AndSize`, but does not store the size.
+   Similar to :c:func:`PyUnicode_AsUTF8AndSize(unicode, NULL)
+   <PyUnicode_AsUTF8AndSize>`, but does not store the size.
```

Comment:
`PyUnicode_AsUTF8AndSize(unicode, NULL)` does not store size either.

Maybe just say that it is equivalent to `PyUnicode_AsUTF8AndSize(unicode, NULL)`? And no more explanations will be needed.